webpack.config.js

```diff
@@ -12,6 +12,32 @@ Encore
     .addEntry('backend', './core-bundle/assets/backend.js')
 ;
 
-const coreConfig = Encore.getWebpackConfig();
+const jsConfig = Encore.getWebpackConfig();
 
-module.exports = [coreConfig];
+Encore.reset();
+
+Encore
+    .setOutputPath('core-bundle/contao/themes/flexible')
+    .setPublicPath('/system/themes/flexible')
+    .setManifestKeyPrefix('')
+    .cleanupOutputBeforeBuild(['*.css', '*.json'])
+    .disableSingleRuntimeChunk()
+    .enableSourceMaps()
+    .enableVersioning()
+    .configureCssLoader(config => {
```

Comment:
Do we need to add the (probably transient dependency) `css-loader` to the `package.json`?

Reply:
Added in 7e826e1124001e2768883da5409c8421fa4e454f.